docs/internationalization/index.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Internationalization (i18n)
+sidebar_position: 10
+---
+
+The following page describes core translation concepts. We use [FormatJS](https://formatjs.io/) for translations. These design principles were considered:
+
+-   Translations should be updatable without any changes to the codebase
+-   [Application workflow](https://formatjs.io/docs/getting-started/application-workflow) from FormatJS
+-   Independence from any translation vendor
+
+## Masking strings
+
+All strings that should be translated must be masked using [`<FormattedMessage />`](https://formatjs.io/docs/react-intl/components#formattedmessage) or, if not possible [`intl.formatMessage()`](https://formatjs.io/docs/react-intl/api#formatmessage). To ensure that all strings are translated we use [`@calm/eslint-plugin-react-intl`](https://www.npmjs.com/package/@calm/eslint-plugin-react-intl).
+
+A stable and unique ID must be used to identify the string. IDs must be prefixed with a project identifier to prevent clashes with other translations. IDs must be named camelCase with dots representing hierarchy.
+
+:::caution
+
+Translations are lost when an ID changes. Changing an existing ID should be well considered.
+
+:::
+
+Examples:
+
+```ts
+intl.formatMessage({ id: "cometDemo.menu.dashboard", defaultMessage: "Dashboard" });
+intl.formatMessage({ id: "cometDemo.menu.pageTree.mainNavigation", defaultMessage: "Main navigation" });
+```
+
+## Extracting strings
+
+All masked strings are extracted using `intl:extract` and pushed to a translation repository. This repository is used to work with the translation vendor. The CI/CD should ensure that changes to the extracted messages are pushed to the translation repository. The extracted strings are pushed from the translation repository to the translation vendor. Updated translations should be synced from the translation vendor to the translation repository (e.g. by using a repository integration).
+
+## Including translations
+
+All strings from all languages should be pulled from the translation repository to the application codebase. Currently, this can be accomplished using `intl:update`.
```

Comment:
```suggestion
All strings in all languages should be pulled from the translation repository to the application codebase. Currently, this can be accomplished using `intl:update`.
```

I'm not completely sure about this preposition. In my opinion it could be `in` or `for`, but I don't think `from` is correct.